### ISSUE TITLE
Will this allow my company to achieve Quantum Supremacy?

### ISSUE THREAD

**Issue by xaviervia** [joaomilho/Enterprise#41]
Management is worried that the company is going to lose its competitive edge now that Google announced Quantum Supremacy. 

I see Enterprise™ is quantum according to the keywords, but how can I leverage it going forward so that my customers can experience its disrupting speed?

**Comment by joaomilho** [comment]
Yes. Enterprise is non-deterministic out of the box and this is what 99% of humans understand as being Quantum. Therefore, Enterprise is Quantum-out-of-the-box. Updating the description soon.
